server/compute/diffexp_cxg.py

```diff
@@ -186,7 +181,7 @@ def fp_err_set(err, flag):
 
 
 def mean_var_cnt_dense(matrix, _, rows):
-    xslc = matrix.multi_index[rows, :]
+    xslc = matrix.multi_index[rows]
```

Comment:
The dense matrix is still 2-dim, but the 2nd dim can be elided, right?

Reply:
Correct!